src/dotnet/APIView/ClientSPA/src/app/_helpers/router-helpers.ts

```diff
@@ -1,6 +1,8 @@
 import { ActivatedRoute } from "@angular/router";
 import { SCROLL_TO_NODE_QUERY_PARAM } from "./common-helpers";
 
+export const REVIEW_PAGE_ROUTE_REGEX = "\/[a-z]+?\/[a-zA-Z0-9]+";
```

Comment:
this is like a generic route. Is there anything specific for the `review` ones?

Reply:
I have removed this. I can access the page name from the route directly so I don't need this.